dd-sdk-android/src/test/kotlin/com/datadog/android/rum/assertj/ViewEventAssert.kt

```diff
@@ -267,9 +267,63 @@ internal class ViewEventAssert(actual: ViewEvent) :
         return this
     }
 
+    fun hasCpuMetric(expectedTicks: Double?): ViewEventAssert {
+        assertThat(actual.view.cpuTicksCount)
+            .overridingErrorMessage(
+                "Expected RUM event to have view.cpu_ticks_count $expectedTicks " +
+                    "but was ${actual.view.cpuTicksCount}"
+            )
+            .isEqualTo(expectedTicks)
+
+        val expectedTicksPerSeconds = if (expectedTicks != null) {
+            (expectedTicks * ONE_SECOND_NS.toDouble()) / actual.view.timeSpent.toDouble()
```

Comment:
very minor: probably `toDouble` calls are not  needed here, `expectedTicks` is already `Double`